AppCenterCrashes/AppCenterCrashes/Internals/Util/MSApplicationForwarder.m

```diff
@@ -24,7 +24,7 @@ static BOOL isApplicationForwarderEnabled() {
 static NSString *const kMSCrashOnExceptionsKey = @"NSApplicationCrashOnExceptions";
 
 static BOOL isCrashOnExceptionsEnabled() {
-  NSNumber *crashOnExceptions = [MS_USER_DEFAULTS objectForKey:kMSCrashOnExceptionsKey];
+  NSNumber *crashOnExceptions = [[NSUserDefaults standardUserDefaults] objectForKey:kMSCrashOnExceptionsKey];
```

Comment:
Is this expected change?

Reply:
Yes. Our implementation `MSUserDefaults` will automatically append "MSAC" to any given key now (after all, it's our overload of `NSUserDefaults` so this is in fact one of the things it's expected to do). This key here (`NSApplicationCrashOnExceptions`), however, is a system key, so I use system `NSUserDefaults` for it.

Reply:
Got it. I actually don't like to add a prefix in the methods in `MSUserDefaults` but didn't comment that based on the usage in this PR. Now we might want to consider adding `MSAC` in all constants instead of appending it in the methods. Better to have the prefix explicitly than implicitly.

Reply:
I also thought about it but couldn't come to a conclusion. Let's post a poll

Reply:
Based on the poll, I'm keeping this as is for now. Maybe more people will vote